diff -ruN --exclude '*~' A/doom-modeline.el B/doom-modeline.el
--- A/doom-modeline.el	2022-03-09 00:25:44.333169404 -0500
+++ B/doom-modeline.el	2022-03-09 00:27:16.393206479 -0500
@@ -92,7 +92,7 @@
   '(objed-state misc-info persp-name battery grip irc mu4e gnus github debug repl lsp minor-modes input-method indent-info buffer-encoding major-mode process vcs checker))
 
 (doom-modeline-def-modeline 'minimal
-  '(bar matches buffer-info-simple)
+  '(bar modals matches buffer-info-simple)
   '(media-info major-mode))
 
 (doom-modeline-def-modeline 'special
@@ -104,7 +104,7 @@
   '(misc-info battery irc mu4e gnus github debug minor-modes input-method major-mode process))
 
 (doom-modeline-def-modeline 'dashboard
-  '(bar window-number buffer-default-directory-simple)
+  '(bar window-number modals buffer-default-directory-simple)
   '(misc-info battery irc mu4e gnus github debug minor-modes input-method major-mode process))
 
 (doom-modeline-def-modeline 'vcs
@@ -112,15 +112,15 @@
   '(misc-info battery irc mu4e gnus github debug minor-modes buffer-encoding major-mode process))
 
 (doom-modeline-def-modeline 'package
-  '(bar window-number package)
+  '(bar window-number modals package)
   '(misc-info major-mode process))
 
 (doom-modeline-def-modeline 'info
-  '(bar window-number buffer-info info-nodes buffer-position parrot selection-info)
+  '(bar window-number modals buffer-info info-nodes buffer-position parrot selection-info)
   '(misc-info buffer-encoding major-mode))
 
 (doom-modeline-def-modeline 'media
-  '(bar window-number buffer-size buffer-info)
+  '(bar window-number modals buffer-size buffer-info)
   '(misc-info media-info major-mode process vcs))
 
 (doom-modeline-def-modeline 'message
@@ -128,7 +128,7 @@
   '(objed-state misc-info battery debug minor-modes input-method indent-info buffer-encoding major-mode))
 
 (doom-modeline-def-modeline 'pdf
-  '(bar window-number matches buffer-info pdf-pages)
+  '(bar window-number modals matches buffer-info pdf-pages)
   '(misc-info major-mode process vcs))
 
 (doom-modeline-def-modeline 'org-src
